active/Expander/Expander.md

```diff
@@ -0,0 +1,173 @@
+
+## Events
+| Name | Description | 
+| :---------- | :------- | 
+| Expanding | Occurs when expanded |
+| Collapsed| Occurs when collapsed |
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+~~~~
+enum ExpandDirection
+{
+    Down = 0,
+    Up = 1
+};
+
+runtimeclass ExpanderExpandingEventArgs
+{
+}
+
+runtimeclass ExpanderCollapsedEventArgs
+{
+}
+ 
+unsealed runtimeclass Expander : Windows.UI.Xaml.Controls.ContentControl
+{
+    Expander();
+
+    Object Header{ get; set; };
+    Windows.UI.Xaml.DataTemplate HeaderTemplate{ get; set; };
+    Windows.UI.Xaml.Controls.DataTemplateSelector HeaderTemplateSelector{ get; set; };
+
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean IsExpanded{ get; set; };
+
+    [MUX_DEFAULT_VALUE("winrt::ExpandDirection::Down")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    ExpandDirection ExpandDirection{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Expander, ExpanderExpandingEventArgs> Expanding;
+    event Windows.Foundation.TypedEventHandler<Expander, ExpanderCollapsedEventArgs> Collapsed;
+
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateSelectorProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty IsExpandedProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ExpandDirectionProperty{ get; };
+}
+~~~~
+
+## Theme Resources
+
+| Name| Description | 
+| :---------- | :------- | 
+| ExpanderHeaderBackground | Header background color| 
+| ExpanderChevronBackground | Chevron background color| 
+| ExpanderChevronForeground | Chevron foreground color| 
+| ExpanderChevronMargin | Chevron margin thickness| 
+| ExpanderChevronGlyph | Chevron glyph| 
+| ExpanderChevronWidth | Chevron width|
+| ExpanderPopinVerticalOffset | vertical offset for animation| 
+
+# Inputs and Accessibility
+## UI Automation Patterns
+Expander will use a ExpandCollapsePattern. Expanding/Collapsing the expander will raise [RaisePropertyChanged](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationpeer.raisepropertychangedevent?view=winrt-19041) with the property changed being the [ExpandCollapseProperty](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.expandcollapsepatternidentifiers.expandcollapsestateproperty?view=winrt-19041) property.
+
+## Keyboard
+* Tabbing brings focus to the Header, neither the entire Expander nor the chevron is a tab-stop
+* When focus is on the Header, space key expands and collapse and focus does not move
+* Keyboard navigation inside the Expander's Header/Content is based on the content inside them
+
+## GamePad
+The Expander can be expanded and collapsed with A. Spatial navigation will navigate between the content of the Expander. 
+
+## Narrator
```

Comment:
Just Narrator or any screen reader?

Reply:
I'll update this area for any screen reader